test/e2e/auth/pod_security_policy.go

```diff
@@ -19,7 +19,7 @@ package auth
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
goimports does this now for performance reasons since the path uses a symlink and the import path isnt the real path.

Reply:
Ah, this is good to know. Always thought it was a vscode thing

Reply:
You're the 2nd person to suggest that to me but I tested manually with goimports and confirmed its not IDE based.